### PR TITLE
feat(mac): :sparkles: use quick look for image popups

### PR DESCRIPTION
**Pull request by piotrski**
Use system quick look to preview images instead of custom popup

image

### CODE REVIEW

**Comment by barryanders** [comment]
Perfect!

**Comment by Leland** [comment]
Huge win.

A major next step would be making the images focusable, so users can <kbd>Tab</kbd> between them and press <kbd>Space</kbd> to trigger QuickLook views.

**Comment by divamgupta** [comment]
Thanks for the PR. I have plans to build it for windows and other OS as well. Can we only use quick look when its macOS

**Comment by piotrski** [comment]
This is exactly what I did, I check the platform and use quick look only on mac, all others use the old popup window.

> Thanks for the PR. I have plans to build it for windows and other OS as well. Can we only use quick look when its macOS

File: electron_app/src/components/History.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <div  class="animatable_content_box ">
     
         <div v-if="Object.values(app_state.history).length > 0">
-            <div v-for="history_box in Object.values(app_state.history).reverse()" :key="history_box.key" style="clear: both;">
+            <div v-for="history_box in Object.values(app_state.history)" :key="history_box.key" style="clear: both;">
```
Comment (divamgupta) [inline electron_app/src/components/History.vue:6]: why remove the .reverse ?

Comment (piotrski) [reply electron_app/src/components/History.vue:6]: I must have made a mistake while rebasing

Comment (piotrski) [reply electron_app/src/components/History.vue:6]: BTW why history is stored as an object with random keys instead of array?

**Comment by Leland** [comment]
Fixes #173

**Pull request closed without merging**